image.yaml

```diff
@@ -1,25 +1,62 @@
-- name: operator-builder
-  version: 1.0.0
-  from: golang:1.14
-  description: Builder Image for the Operator
+- schema_version: 1
+  name: "operator-builder"
+  version: "1.0.0"
+  from: "registry.redhat.io/ubi8/go-toolset:1.14.12"
+  description: "Golang builder image for the RHPAM Kogito Operator"
 
   modules:
     repositories:
       - path: modules
     install:
-      - name: org.kie.kogito.goModDownload
-      - name: org.kie.kogito.buildOperator
-- name: operator-runtime
-  version: 1.0.0
-  from: registry.access.redhat.com/ubi8/ubi-minimal:latest
-  description: Runtime Image for the Operator
+      - name: org.kie.kogito.gomoddownloader
+      - name: org.kie.kogito.builder
+
+  osbs:
+    configuration:
+      container:
+        image_build_method: imagebuilder
+        platforms:
+          only:
+            - x86_64
+
+- name: "operator-runtime"
+  version: "1.0.0"
+  from: "registry.redhat.io/ubi8/ubi-minimal:latest"
+  description: "Runtime Image for the RHPAM Kogito Operator"
+
+  labels:
+    - name: maintainer
+      value: bsig-cloud@redhat.com
+    - name: com.redhat.component
+      value: rhpam-7-kogito-rhel8-operator-container
+    - name: io.k8s.description
+      value: Operator for deploying RHPAM Kogito Application
+    - name: io.k8s.display-name
+      value: Red Hat PAM Kogito Operator
+    - name: io.openshift.tags
+      value: rhpam,kogito,operator
 
   artifacts:
-    - name: manager
-      image: operator-builder
-      path: /workspace/manager
-      dest: /
+    - name: kie-cloud-operator
+      image: builder
+      path: /usr/local/bin/kie-cloud-operator
+      dest: /usr/local/bin
+
+  osbs:
+    configuration:
+      container:
+        go:
+          modules:
+            - module: github.com/kiegroup/rhpam-kogito-operator
```

Comment:
This needs to be tested on OSBS.
Our builder image needs to output a tgz with the operator on it;